src/Illuminate/Foundation/helpers.php

```diff
@@ -554,10 +554,16 @@ function mix($path, $manifestDirectory = '')
         $manifest = $manifests[$manifestPath];
 
         if (! isset($manifest[$path])) {
-            throw new Exception(
-                "Unable to locate Mix file: {$path}. Please check your ".
-                'webpack.mix.js output paths and try again.'
-            );
+            $mixMessage = "Unable to locate Mix file: {$path}. Please check your ".
+                          'webpack.mix.js output paths and try again.';
+
+            if (app()->environment() === 'production') {
```

Comment:
I'm wondering if checking for production is the right check? It might make more sense to check if we are in debug mode or not? That future proofs against people in staging environments etc?

So something like `if (app('config')->get('app.debug')) {`